lasagne/nonlinearities.py

```diff
@@ -269,6 +269,75 @@ def elu(x):
     return theano.tensor.switch(x > 0, x, theano.tensor.expm1(x))
 
 
+# selu
+class SELU(object):
+    """
+    Scaled Exponential Linear Unit
+    :math:`\\varphi(x)=\\lambda \\left[(x>0) ? x : \\alpha(e^x-1)\\right]`.
```

Comment:
Just for consistence with the others, could you please remove the period in the end? (It's not a sentence.)